app/src/main/java/org/lineageos/selfie/MainActivity.kt

```diff
@@ -437,6 +437,13 @@ class MainActivity : AppCompatActivity() {
 
     override fun onKeyDown(keyCode: Int, event: KeyEvent?): Boolean {
         return when (keyCode) {
+            KeyEvent.KEYCODE_FOCUS -> {
+                if (event?.repeatCount == 1) {
+                    viewFinderTouchEvent = null
+                    viewFinder.performClick()
+                }
+                true
+            }
             KeyEvent.KEYCODE_VOLUME_UP,
             KeyEvent.KEYCODE_VOLUME_DOWN -> {
```

Comment:
Add KEYCODE_CAMERA here?

Reply:
Well maybe before KEYCODE_VOLUME_UP

Reply:
I don't think we want to handle this kind of behavior for camera button.

Reply:
Hmm, maybe it's ok. I'll try it.

Reply:
Is there an equivalent video camera button? Else it might make sense to also handle that

Reply:
Ok, it works fine.